tox.ini

```diff
@@ -108,10 +108,12 @@ commands =
 [molecule_common]
 changedir = {toxinidir}
 deps =
+    ansible-lint==4.2.0
     {env:LSR_ANSIBLE_DEP:ansible}
     docker{env:LSR_MOLECULE_DOCKER_VERSION:}
     molecule<3
     selinux
+    wcwidth==0.1.9;python_version=="3.6"
```

Comment:
What happens if the python version is lower?

Reply:
not sure - @nhosoi do we need this change for collections?  If not, please remove this change from the commit

Reply:
No, not for collections. sync-template brought it in. All right. Removing the line.